### PR TITLE
improving markup adding styles for mobile

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Is it possible to have the styles in a stylesheet?

File: content/pages/1--about/index.md [hunk 1]
```diff
@@ -8,140 +8,146 @@ type: pages
...
+    }
   }
 
 </style>
+<div class="about-intro">
+  <img src="images/si-logo.svg" class="about-si-logo" alt="Samsung Internet Logo: a white ringed planet on a purple background">
+  <p class="about-si-description">Samsung Internet for Android is the web browser that is pre-installed on Samsung Galaxy phones and tablets. It is based on the open source Chromium project, to which Samsung are a major contributor.</p>
+</div>
 
-<div class="about-content">
```
Comment (AdaRoseCannon) [inline content/pages/1--about/index.md]: Good markup changes :+1:

File: src/layouts/index.js [hunk 2]
```diff
@@ -119,6 +119,12 @@ class Layout extends React.Component {
                   letter-spacing: 0.05em;
                   margin: 0;
                 }
+                h4 {
+                  font-family: ${this.state.font600loaded
```
Comment (AdaRoseCannon) [inline src/layouts/index.js]: Shouldn't this just be in the CSS file?

**Comment by RedRoxProjects** [comment]
Merging as per comments in slack re distribution of CSS for current templates - some future tweaking to be done 🛠

**Pull request merged**